.github/workflows/stale.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: '31 3 * * *' # randomly chosen time of day
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 21
+          days-before-pr-stale: 30
+          days-before-close: 7
+          stale-issue-message: |-
+            This issue seems inactive. If it's still relevant, please add a comment saying so. Otherwise, take no action.
+            → If there's no activity within a week, then a bot will automatically close this.
+            Thanks for helping to improve Shopify's design system and dev experience.
```

Comment:
@jjgali Here is the current comment message displayed to contributors. Do you have any recommendations on how we might improve this and make sure it is received positively?